admin/containers/Settings/Settings.js

```diff
@@ -51,77 +79,75 @@ class Settings extends Component {
         document.body.classList.remove("settings-page");
     }
 
-    setOption(option, value) {
-        this.updatedOptions[option] = value;
-    }
+    setOption = (option, value) => {
+        const { updatedOptions } = this.state;
+        updatedOptions[option] = value;
+    };
 
-    submitData(e) {
+    submitData = e => {
         e.preventDefault();
         const settings = [];
-        Object.keys(this.updatedOptions).forEach(option => {
+        Object.keys(this.state.updatedOptions).forEach(option => {
             settings.push({
                 option,
-                value: this.updatedOptions[option]
+                value: this.state.updatedOptions[option]
             });
         });
         this.props.updateOptions(settings).then(() => {
             notify.show("Site settings saved", "success", 3000);
         });
-    }
+    };
+
+    handleNavClick = page => {
+        this.setState({ selected: page });
+    };
+
     render() {
+        const { selected } = this.state;
+        const { options } = this.props;
         const data = {};
         const { t } = this.context;
-        if (this.props.options.loading) {
+        if (options.loading) {
             return <div>hello</div>;
         }
-        this.props.options.settings.forEach(setting => {
+        options.settings.forEach(setting => {
             data[setting.option] = setting;
         });
 
         return (
             <section className="module-xs">
                 <div className="masonry-grid">
                     <div className="card masonry-grid-item">
+                        <NavWrapper>
```

Comment:
Please remove `<div className="masonry-grid">` and the class `masonry-grid-item`